src/_posts/compliance/2000-01-01-hds.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Hébergeur de Données de Santé - Health Data Hosting Provider 
+nav: HDS
+modified_at: 2022-09-12 00:00:00
+tags: compliance hds
+---
+
+*(version française [ci-dessous](./2000-01-01-hds.md#fr-has-scalingo-an-hds-certification))*
```

Comment:
404 on the link https://documentation-service-pr1739.osc-fr1.scalingo.io/compliance/hds